modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,108 @@
 
 [id="understanding-upgrade-channels_{context}"]
 
-= Upgrade channels and release paths
-Cluster administrators can configure the upgrade channel from the web console.
-
-
-[id="candidate-version-channel_{context}"]
-== candidate-{product-version} channel
-
-The `candidate-{product-version}` channel contains candidate builds for a z-stream ({product-version}.z) and previous minor version releases. Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version of {product-title}. A release candidate is any build that is available in the candidate channel, including ones that do not contain link:https://semver.org/spec/v2.0.0.html#spec-item-9[a pre-release version] such as `-rc` in their names. After a version is available in the candidate channel, it goes through more quality checks. If it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels. Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}` or `stable-{product-version}` channels, it is a Red Hat-supported version. The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
-
-You can use the `candidate-{product-version}` channel to update from a previous minor version of {product-title}.
+= Update channels
 
+ifndef::openshift-origin[]
 [id="fast-version-channel_{context}"]
 == fast-{product-version} channel
-
-The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
-
-You can use the `fast-{product-version}` channel to upgrade from a previous minor version of {product-title}.
-
-ifndef::openshift-origin[]
+The `fast-{product-version}` channel is updated with new versions of {product-title} {product-version} as soon as Red Hat declares the version as a general availability (GA) release. As such, these releases are fully commercially supported and purposed to be used in production environments.
```

Comment:
I added some details in the `supported` vs `commercially supported` thread. `Just `supported` would be the best practice here

Reply:
Ok, I'll give Tim time to respond and then move forward with your suggestion as that's what I had originally but product experience requested the formal terminology.